cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -1636,7 +1636,7 @@ subroutine makemask
       !-----------------------------------------------------------------
 
       bm = c0
-!     uvm = c0
+      uvm = c0
```

Comment:
Similarly here. The HaloUpdate should take care of uvm values in the ghost cells.

Reply:
It aborts at line [1673 in `makemask`]( https://github.com/CICE-Consortium/CICE/blob/a76893a0d47c3c2d0c8fbe83238aa3b000dabd1b/cicecore/cicedynB/infrastructure/ice_grid.F90#L1673)